tv_apps/smart-system/js/browser_context_menu.js

```diff
@@ -155,13 +159,18 @@
     }
     this._injected = true;
     this.buildMenu(menu);
-    this.element.classList.add('visible');
     document.activeElement.blur();
     this.scrollable.catchFocus();
```

Comment:
By earlier discussion  we ignore this temporarily and solve it in other bug due to `mouse-down` and `mouse-up` handling mismatch.